src/arrays/MemoryBinarySearch.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.13;
+pragma solidity ^0.8.18;
```

Comment:
don't think we have to restrict libs to any version, might just specify >0.8.0 in case other repos doesn't upgrade for some reason